src/App.jsx

```diff
@@ -10,10 +10,13 @@ import ForgotPassword from './pages/ForgotPassword/ForgotPassword';
 import ResetPassword from './pages/ForgotPassword/ResetPassword';
```

Comment:
Can you add a eslint rule to order imports starting from 3rd party to your own, in this case to be like:
```
import { QueryClient, QueryClientProvider } from 'react-query';
import 'react-toastify/dist/ReactToastify.css';

import CustomToastContainer from './components/ToastContainer';
import ResetPassword from './pages/ForgotPassword/ResetPassword';
```

Reply:
done, added rules and refactored code